src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs

```diff
@@ -383,7 +383,7 @@ await LogTestHostCreatedAsync(
         // Check if we're in the test host or we should check test controllers extensions
         // Environment variable check should not be needed but in case we will rollback to use only env var we will need it.
         if ((!testHostControllerInfo.HasTestHostController ||
-            systemEnvironment.GetEnvironmentVariable($"{EnvironmentVariableConstants.TESTINGPLATFORM_TESTHOSTCONTROLLER_SKIPEXTENSION}_{testHostControllerInfo.GetTestHostControllerPID(true)}") != "1")
```

Comment:
@Evangelink @MarcoRossignoli FYI, we never pass false to `GetTestHostControllerPID`. Not sure if we should remove that parameter altogether?

Reply:
It's internal so I don't mind removing as we don't need it for now.

Reply:
Merging the PR anyways, let's do a follow-up if @MarcoRossignoli agrees.